<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="register_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>220</width>
     <height>310</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QStackedWidget" name="reg_stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>220</width>
      <height>270</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="register_page">
     <widget class="QLineEdit" name="username_edit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>180</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="password_edit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="realname_edit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="phone_edit">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>180</width>
        <height>30</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="register_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>220</y>
        <width>160</width>
        <height>32</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 11pt &quot;Microsoft YaHei UI&quot;;</string>
      </property>
      <property name="text">
       <string>注册</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="error_page">
     <widget class="QLabel" name="text_lb">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>220</width>
        <height>160</height>
       </rect>
      </property>
      <property name="text">
       <string>提示</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="return_btn">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>140</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="register_lb">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>10</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 12pt &quot;Microsoft YaHei UI&quot;;</string>
    </property>
    <property name="text">
     <string>注册用户</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>10</y>
      <width>25</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
